test/passing/module.ml

```diff
@@ -28,3 +28,65 @@ module C = struct
 
   and B : sig end = B
 end
+
+module M :
+  S
+  with type long_long_type = t
+   and type long_long_long_type = u
+   and type very_long_long_type = t =
+  M
+
+module type X = sig
+  val n : (module S with type t = t)
+
+  val o :
+    (module S
+         with type long_long_type = t
+          and type long_long_long_type = u
+          and type very_long_long_type = t)
+end
+
+module X = (val X.n : S with type t = t)
+
+module Y = ( val X.o : S
+               with type long_long_type = t
```

Comment:
Here there is indentation that depends on width of user-code, the name `Y`. This is something that's considered to be a bug. (Though I think it is possible currently, at least due to interactions with comments.)

Reply:
Fixed